seal.go

```diff
@@ -155,6 +159,9 @@ type KeyCreationParams struct {
 // UpdateKeyPCRProtectionPolicy. This key doesn't need to be stored anywhere, and certainly mustn't be stored outside of the encrypted
 // volume protected with this sealed key file. The key is stored encrypted inside this sealed key file and returned from future calls
 // to SealedKeyObject.UnsealFromTPM.
+//
+// The authorization key can also be chosen and provided by setting
+// AuthKey in the params argument.
```

Comment:
The comment / doc lines don't need to wrap quite so early.

Reply:
I never bothered you with this but the current comments in secboot don't match the wrapping conventions of either snapd or the go std lib:  https://github.com/golang/go/blob/master/src/crypto/crypto.go#L165   The lines here tend to be way too long.  While there is no strong preference for short code lines, reasonable wrapping of comments is typical. It's actually hard for me to follow your long line conventions in my setup.